docs/src/run.md

```diff
@@ -0,0 +1,406 @@
+# Heat Equation + Slab Tutorial
```

Comment:
@LenkaNovak using Literate.jl, we should be able to only have one version of the tutorial file (the one that lives in `experiments/...`) which then generates a file to be used by `Documenter.jl` when the docs are generated. Was there a reason for adding another markdown file?